ibm_catalog.json

```diff
@@ -0,0 +1,239 @@
+                  "grouping": "deployment",
+                  "original_grouping": "deployment"
+              }
+            },
+            {
+              "key": "cluster_resource_group_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster resource group",
+              "required": true,
+              "custom_config": {
+                  "type": "resource_group",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment",
+                  "config_constraints": {
+                      "identifier": "rg_id"
+                  }
+              }
+            },
+            {
+              "key": "cloud_monitoring_access_key"
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace"
+            },
+            {
+              "key": "cloud_monitoring_agent_tags"
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations"
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type"
+            },
+            {
+              "key": "cloud_monitoring_instance_region",
+              "display_name": "Region",
+              "custom_config": {
+                "type": "vpc_region",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "generationType": "2"
+                }
+              }
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter"
+            },
+            {
+              "key": "cloud_monitoring_secret_name"
+            },
+            {
+              "key": "cluster_config_endpoint_type"
+            },
+            {
+              "key": "cluster_data"
+            },
+            {
+              "key": "cloud_logs_ingress_endpoint"
+            },
+            {
+              "key": "cloud_logs_ingress_port"
+            },
+            {
+              "key": "is_vpc_cluster"
+            },
+            {
+              "key": "logs_agent_additional_log_source_paths"
+            },
+            {
+              "key": "logs_agent_additional_metadata"
+            },
+            {
+              "key": "logs_agent_enable_scc"
+            },
+            {
+              "key": "logs_agent_enabled"
+            },
+            {
+              "key": "logs_agent_exclude_log_source_paths"
+            },
+            {
+              "key": "logs_agent_iam_api_key"
+            },
+            {
+              "key": "logs_agent_iam_environment"
+            },
+            {
+              "key": "logs_agent_iam_mode"
```

Comment:
This should be a dropdown picker with the supported values. Looks like variable validation is also missing in the DA for it too. Only allowed value are `TrustedProfile` or `IAMAPIKey`.